LukeMason33/Rotten-Tomatillos
# Feature header dropdown

### Is this a fix or a feature?

Feature

### What is the change/fix?

This adds a filter dropdown menu that allows a user to filter movies by genre.

### Where should the reviewer start?

In `App.js`: review the `filterHandler` function, and where its being passed to the header. 
In `/header-components/filter-menu-component`, review how the movie genres are being filtered from all movies data, and how each option is being created in the `HeaderFilter` component.

### How should this be tested?

In browser, select a genre from the dropdown menu to ensure that only movies which contain those genres are being displayed. Additionally, ensure that selecting the 'All' option will return the user to all movies veiw.
